mantidproject/fitbenchmarking
# Runtime tables are now as expected

#### Description of Work

Fixes #96 
Runtime tables now display the proper results.

#### Testing Instructions

1. In the `mantidpython` console, run `example_runScripts.py`
2. Check if the output tables are as expected.
